docs/en/ros2/px4_ros2_control_interface.md

```diff
@@ -400,6 +401,134 @@ _goto_setpoint->update(
   max_heading_rate_rad_s);
 ```
 
+#### Fixed-Wing Lateral and Longitudinal Setpoint (FwLateralLongitudinalSetpointType)
+
+<Badge type="warning" text="Fixed wing only" />
+
+::: info
+This setpoint type is only supported for fixed-wing vehicles.
+:::
+
+Use the [`px4_ros2::FwLateralLongitudinalSetpointType`](https://auterion.github.io/px4-ros2-interface-lib/classpx4__ros2_1_1FwLateralLongitudinalSetpointType.html) to directly control the lateral and longitudinal dynamics of a fixed-wing vehicle.
+This setpoint is streamed to the PX4 _FwLateralLongitudinalControl_ module, which decouples lateral and longitudinal inputs while ensuring that vehicle limits are respected.
+
+To control the vehicle, at least one lateral **and** one longitudinal setpoint must be provided:
+
+1. Of the longitudinal inputs: either `altitude` or `height_rate` must be finite to control vertical motion.
+   If both are set to `NAN`, the vehicle will maintain its current altitude.
+2. Of the lateral inputs: at least one of `course`, `airspeed_direction`, or `lateral_acceleration` must be finite.
+
+For a detailed description of the controllable parameters, please refer to message definitions ([FixedWingLateralSetpoint](../msg_docs/FixedWingLateralSetpoint.md) and [FixedWingLongitudinalSetpoint](../msg_docs/FixedWingLongitudinalSetpoint.md)).
+
+##### Basic Usage
+
+In the simplest case, you can provide only a `course` and an `altitude` setpoint:
+
+```cpp
+const float altitude_msl = 500.F;
+const float course = 0.F; // due North
+_fw_lateral_longitudinal_setpoint->updateWithAltitude(altitude_msl, course);
+```
+
+From these setpoints, PX4 will compute the required _roll angle_, _pitch angle_ and _throttle_ setpoints that are sent to lower level controllers.
+This is done as follows:
+
+- Lateral control output:
+
+  course setpoint (set by user) &rarr; airspeed direction (heading) setpoint &rarr; lateral acceleration setpoint &rarr; roll angle setpoint.
+- Longitudinal control output:
+
+  altitude setpoint (set by user) &rarr; height rate setpoint &rarr; pitch angle setpoint and throttle settings.
+
+Alternatively, you can provide only a course and a height rate setpoint:
```

Comment:
```suggestion
Alternatively, you can provide only a course and a `height_rate` setpoint:
```

Reply:
Why would I want to do this?

Reply:
I think so. Suggestion here https://github.com/PX4/PX4-Autopilot/pull/24785/files#r2101457965